### PR TITLE
Update TF version in CI

### PR DESCRIPTION
**Pull request by GoGstickGo**
Fix: #11

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -91,6 +91,7 @@ jobs:
           - 0.13.6
           - 0.14.8
           - 0.15.0-beta2
```
Comment (mlech-reef) [inline .github/workflows/ci.yml:93]: Please remove

**Review by mlech-reef — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (GoGstickGo) [reply .github/workflows/ci.yml]: Roger that.

**Comment by ppolewicz** [comment]
@mlech-reef CI is failing, but I don't understand why

**Comment by mlech-reef** [comment]
> @mlech-reef CI is failing, but I don't understand why

The CI secrets are not inherited by the forks because of obvious security reasons. In SDK and CLI, the integration tests are skipped but not here.

**Review by mlech-reef — APPROVED** [review]

**Comment by mlech-reef** [comment]
@GoGstickGo please rebase. I will test your changes manually.

**Comment by GoGstickGo** [comment]
Roger that!

**Comment by GoGstickGo** [comment]
Can u merge the  PR ? @mlech-reef

**Comment by ppolewicz** [comment]
I see CI has failed, but I think it may have been caused by a bug that was fixed recently, so I re-ran it now, lets see if it works this time.

**Comment by ppolewicz** [comment]
@mszumocki-reef please look into it - perhaps the fixed CI needs to be merged here (or rebased)?

**Comment by mlech-reef** [comment]
@GoGstickGo @ppolewicz I have merged the `master` branch. Now everything should be fine. I will wait for CI to finish and merge.

**Pull request merged**